src/content/docs/usage/installation.md

```diff
@@ -83,86 +70,86 @@ The main steps will be the following:
+### Edge releases
 
-### Updating Nextflow
+Nextflow has two `stable` releases per year and monthly `edge` releases.
+Some pipelines may require the use of Nextflow `edge` releases in order to exploit _cutting edge_ features.
 
-We recommend using a personal installation of Nextflow where possible, instead of using a system-wide installation. This makes it easier to update.
+Nextflow installs the latest stable version by default.
+You can get an edge release either by defining the exact version with `NXF_VER`
+or by using the `NXF_EDGE` environment variable:
 
-Updating nextflow is as simple as running `nextflow self-update`
-or `conda update nextflow`, depending on how it was installed.
+```bash
+NXF_VER=24.02.0-edge nextflow run ...
+NXF_EDGE=1 nextflow self-update
+```
 
 ## Pipeline software
 
 An analysis pipeline chains the execution of multiple tools together.
 Historically, all tools would have to be manually installed — often a source of great frustration and a key step where reproducibility between analyses is lost.
 nf-core pipelines utilise the built-in support for software packaging that Nextflow offers: all can work with Docker and Singularity, and most pipelines also support Conda.
 
+To use any of the below, simply run your nf-core pipeline with `-profile <type>`.
+For example, `-profile docker` or `-profile conda`.
+
 - [Docker](https://docs.docker.com/install/)
   - Typically used locally, on single-user servers, and the cloud
   - Analysis runs in a _container_, which behaves like an isolated operating system
-  - Previously required system root access, though a "rootless mode" has been available since 2019
+  - Typically requires system root access, though a _"rootless mode"_ is available
 - [Singularity](https://www.sylabs.io/)
-  - Often used as an alternative to Docker on multi-user systems, such as HPC systems
-  - Also runs _containers_ and can create these from Docker images
-  - Does not need root access or any daemon processes - images are built from files
-- [Podman](https://podman.io/)
-  - Often used as an alternative to Docker on multi-user systems, such as HPC systems
-  - Is a daemonless container engine that can serve as a drop-in replacement for Docker
-- [Charliecloud](https://hpc.github.io/charliecloud/)
-  - Often used as an alternative to Docker on multi-user systems, such as HPC systems
-  - Uses Linux user namespaces to run containers with no privileged operations or daemons
-- [Shifter](https://www.nersc.gov/research-and-development/user-defined-images/)
-  - An experimental implementation of a container system that can convert from a wide range of other images
+  - Often used as an alternative to Docker on HPC systems
+  - Also runs _containers_, and can optionally create these from Docker images
+  - Does not need root access or any daemon processes
+- [Apptainer](https://apptainer.org/)
```

Comment:
I wonder if this should go before singularity nowadays, but not important

Reply:
I feel like more people are familiar with the term Singularity still, so would err on having it like this for now personally. But not a strong opinion.